### PR TITLE
Check individual characteristics

### LINKED ISSUE #182: getPopulationResultsValues removes columns when results are NaN
I have a simulation that produces NaN. This is an error with my simulation for example
When I am trying to return the value for a specific output and individual Id, I get NaN

```
populationResults$getValuesForIndividual("Organism|PeripheralVenousBlood|Caffeine|Plasma (Peripheral Venous Blood)", 2)

[1] NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN
 [37] NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN
 [73] NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN
[109] NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN NaN
````

However, when I call the getPopulationResultsValues with the same arguments, I get NULL

I think this is a bug
```
getPopulationResultsValues(populationResults, "Organism|PeripheralVenousBlood|Caffeine|Plasma (Peripheral Venous Blood)", 2)
NULL
```

The reason is I believe this weird filtering happening in the method here
https://github.com/Open-Systems-Pharmacology/OSPSuite-R/blob/develop/R/utilities-simulation-results.R#L116

### PR DESCRIPTION
**Pull request by PavelBal**
Fixes #182

Also fixes https://github.com/esqLABS/esqlabsR/pull/183

### CODE REVIEW

File: R/create-individual.R [hunk 1]
```diff
@@ -111,12 +111,16 @@ createIndividualCharacteristics <- function(species,
...
+  if (species == Species$Human && !enumHasKey(key = population, enum = HumanPopulation)){
+    stop(messages$errorWrongPopulation(species, population))
+  }
   individualCharacteristics$population <- population
   individualCharacteristics$gender <- gender
-  individualCharacteristics$age <- createSnapshotParameter(age, ageUnit)
-  individualCharacteristics$weight <- createSnapshotParameter(weight, weightUnit)
-  individualCharacteristics$height <- createSnapshotParameter(height, heightUnit)
-  individualCharacteristics$gestationalAge <- createSnapshotParameter(gestationalAge, gestationalAgeUnit)
+  individualCharacteristics$age <- .createSnapshotParameter(age, ageUnit)
```
Comment (PavelBal) [inline R/create-individual.R:120]: Added a `.` for internal function name.

File: R/messages.R [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#' List of functions and strings used to signal error messages
+#' Extends the `messages` list from ospsuite.utils
+messages <- ospsuite.utils::messages
```
Comment (PavelBal) [inline R/messages.R:3]: We now have "internal" messages list that is build from the utils-list.

File: R/snapshot-parameter.R [hunk 3]
```diff
@@ -56,8 +56,8 @@ SnapshotParameter <- R6::R6Class(
   )
 )
 
-createSnapshotParameter <- function(value, unit) {
-  if (is.null(value)) {
+.createSnapshotParameter <- function(value, unit) {
```
Comment (PavelBal) [inline R/snapshot-parameter.R:59]: @msevestre This is what you proposed here, right?
https://github.com/esqLABS/esqlabsR/pull/183

Comment (PavelBal) [reply R/snapshot-parameter.R:59]: Definitely a better solution than converting all nas to NULLs by hand.

**Review by IndrajeetPatil — APPROVED** [review]

File: R/snapshot-parameter.R (hunk 3 above)
Comment (msevestre) [reply R/snapshot-parameter.R:59]: yes

File: tests/testthat/test-create-individual.R [hunk 4]
```diff
@@ -68,18 +99,6 @@ test_that("It sets a random seed if not specified", {
...
-    species = Species$Human,
-    weight = 60,
-    age = 15,
-    gender = Gender$Female
-  )
-  expect_that(createIndividual(individualCharacteristics = human), throws_error())
-})
-
 test_that("It can create reating a human with weight missing", {
   skip_on_os("linux") # TODO enable again as soon as createIndividual/createPopulation runs under Linux
```
[the remark this answers is not in the document]
Comment (PavelBal) [reply tests/testthat/test-create-individual.R]: @msevestre Added a test for `NA`.

**Pull request merged**